docs/platforms/javascript/common/install/late-initializtion.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: Late Initialization (ESM or CJS)
+sidebar_order: 12
+description: "Learn about running Sentry in an ESM or CJS application, in scenarios where you cannot run init early."
+supported:
+  - javascript.node
+  - javascript.connect
+  - javascript.express
+  - javascript.fastify
+  - javascript.hapi
+  - javascript.koa
+---
+
+<Alert>
+  Are you unsure if you should use this installation method? Review our
+  [installation methods](../).
+</Alert>
+
+In order for auto-instrumentation to work, it is generally required to run `Sentry.init()` as early as possible, before anything else is imported in your application.
+
+However, in some cases this may not be possible to do - for example, if you are fetching your DSN from an external source. In this case, you can use the `@sentry/node/preload` hook to ensure modules are wrapped early, which allows you to call `Sentry.init()` later at a time of your choosing.
+
+<Note>
+  We recommend to only use this method if strictly necessary. In most cases, it
+  is better to find a way to run `Sentry.init()` early in your application, in
+  order to ensure that no error can go unreported.
+</Note>
+
+## Late Initialization with CommonJS (CJS)
+
+In your CJS application, use the `@sentry/node/preload` hook with `--require` to ensure modules are wrapped early:
+
+```bash
+# Note: This is only available for Node v18.19.0 onwards.
```

Comment:
q: Is that true for CJS?

Reply:
Actually I don't know, good question - here it says 1.x, so probably it's fine xD https://nodejs.org/api/cli.html#-r---require-module

So I'll remove this here!